### PR TITLE
Tm 180 export application details csv

### PR DESCRIPTION
**Pull request by sscaife**

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Api/SFA.DAS.LevyTransferMatching.Api.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <NoWin32Manifest>true</NoWin32Manifest>
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Api/SFA.DAS.LevyTransferMatching.Api.csproj:5]: Did you mean to change this?

File: src/SFA.DAS.LevyTransferMatching/Models/Application.cs [hunk 2]
```diff
@@ -32,5 +34,11 @@ public class ApplicationLocation
             public int Id { get; set; }
             public int PledgeLocationId { get; set; }
         }
+        public List<PledgeLocation> PledgeLocations { get; set; }
+        public IEnumerable<Sector> PledgeSectors { get; set; }
+        public IEnumerable<Level> PledgeLevels { get; set; }
+        public IEnumerable<JobRole> PledgeJobRoles { get; set; }
+        public string AboutDetails { get; set; }
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching/Models/Application.cs:41]: Why not `Details`?

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.LevyTransferMatching.Api/SFA.DAS.LevyTransferMatching.Api.csproj (hunk 1 above)
Comment (sscaife) [reply src/SFA.DAS.LevyTransferMatching.Api/SFA.DAS.LevyTransferMatching.Api.csproj:5]: I did not, dunno how I missed that. I have no idea why that was added, or how I missed it with 3 changed files. Sorry about that

**Review by chrisfoster76 — APPROVED** [review]
Minor comments

**Review by ConorTill — APPROVED** [review]

**Pull request merged**